iroha_data_model/src/lib.rs

```diff
@@ -545,6 +572,22 @@ pub mod account {
                 metadata: Metadata::default(),
             }
         }
+
+        /// Checks the length of the id is in a valid range
+        ///
+        /// # Errors
+        /// Fails if limit check fails
+        pub fn validate_len(&self, limits: LengthLimits) -> Result<()> {
+            let range: Range<usize> = limits.into();
```

Comment:
Api can be changed to `Into<RangeInclusive<usize>>` to be more abstract

Reply:
Agree. Done